mnist/Dockerfile.model

```diff
@@ -5,5 +5,5 @@ FROM tensorflow/tensorflow:1.15.2-py3
 ADD model.py /opt/model.py
 RUN chmod +x /opt/model.py
 
-ENTRYPOINT ["/usr/bin/python"]
+ENTRYPOINT ["/usr/bin/python3"]
```

Comment:
Should we change this to just be "python"

It looks like that is pointing to the correct version.

```
root@jlewi-tf2:/# which python
/usr/local/bin/python
root@jlewi-tf2:/# python --version
Python 3.6.9
```

It looks like that is what the notebook mnist_gcp.ipynb is doing which is why it doesn't hit the problem.

Reply:
Double-checked and the [tensorflow base image actually creates a symlink](https://github.com/tensorflow/tensorflow/blob/e5bf8de410005de06a7ff5393fafdf832ef1d4ad/tensorflow/tools/dockerfiles/dockerfiles/cpu.Dockerfile#L45) to the current python binary as `/usr/local/bin/python` regardless if that is python version 2 or version 3. So just changing this from `/usr/bin/python` to `/usr/local/bin/python` should be a more generic fix. I've updated the commit to clarify this as well as changing the python path in the ENTRYPOINT. 

Let me know if this is more suitable. 😄